charts/cluster-scanner/templates/_helpers.tpl

```diff
@@ -112,6 +112,16 @@ ise_js_consumer_ack_wait: "240s"
 ise_js_consumer_max_deliver: "1"
 ise_js_consumer_deliver_policy_all: "true"
 ise_js_producer_subject: "analysis.sboms"
+{{- if .Values.runtimeStatusIntegrator.natsJS.tls.enabled }}
+ise_js_tls_skip_verify: {{ .Values.imageSbomExtractor.natsJS.tls.skipVerify | default "false" | quote }}
+{{- if not .Values.imageSbomExtractor.natsJS.tls.skipVerify }}
+{{- if eq (include "cluster-scanner.nats.tls.customTLS" .) "true" }}
```

Comment:
why are you comparing this with a string? doesn't `(include "cluster-scanner.nats.tls.customTLS" .)` return a bool? Mayber we should also rename the definition as `cluster-scanner.nats.tls.hasCustomCert` or `cluster-scanner.nats.tls.customCertDefined`. wdyt?

Reply:
I agree @mech-pig